diff -uNr old-zipios++-0.1.5.9+cvs.2007.04.28/src/appendzip.cpp zipios++-0.1.5.9+cvs.2007.04.28/src/appendzip.cpp
--- old-zipios++-0.1.5.9+cvs.2007.04.28/src/appendzip.cpp	2000-08-25 21:26:26.000000000 +0200
+++ zipios++-0.1.5.9+cvs.2007.04.28/src/appendzip.cpp	2011-05-08 11:41:48.385373131 +0200
@@ -5,6 +5,8 @@
 #include "zipios++/ziphead.h"
 #include "zipios++/zipheadio.h"
 
+#include <cstdlib>
+
 using namespace std ;
 using namespace zipios ;
 
