diff -uNr old-zipios++-0.1.5.9+cvs.2007.04.28/zipios++/meta-iostreams.h zipios++-0.1.5.9+cvs.2007.04.28/zipios++/meta-iostreams.h
--- old-zipios++-0.1.5.9+cvs.2007.04.28/zipios++/meta-iostreams.h	2003-01-04 19:13:03.000000000 +0100
+++ zipios++-0.1.5.9+cvs.2007.04.28/zipios++/meta-iostreams.h	2011-05-08 11:39:00.441373326 +0200
@@ -7,11 +7,7 @@
 
 #include <iostream>
 #include <fstream>
-
-#if defined (HAVE_STD_IOSTREAM) && defined (USE_STD_IOSTREAM)
 #include <sstream>
-#else
 #include <strstream>
-#endif
 
 #endif
diff -uNr old-zipios++-0.1.5.9+cvs.2007.04.28/zipios++/zipios_defs.h zipios++-0.1.5.9+cvs.2007.04.28/zipios++/zipios_defs.h
--- old-zipios++-0.1.5.9+cvs.2007.04.28/zipios++/zipios_defs.h	2000-08-23 20:23:18.000000000 +0200
+++ zipios++-0.1.5.9+cvs.2007.04.28/zipios++/zipios_defs.h	2011-05-08 11:39:45.739373337 +0200
@@ -1,10 +1,13 @@
 #ifndef ZIPIOS_DEFS_H
 #define ZIPIOS_DEFS_H
 
+#include "zipios++/zipios-config.h"
+#include <stdint.h>
+
 namespace zipios {
 
-typedef unsigned short uint16 ;
-typedef unsigned long  uint32 ;
+typedef uint16_t uint16 ;
+typedef uint32_t uint32 ;
 
 } // namespace
 
